python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
+    tap:principal: 1
+    tap:column_index: 113
   - name: refFwhm
     "@id": "#Object.refFwhm"
     datatype: double
     description: Estimated FWHM on the reference band at source center assuming a Gaussian profile
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 114
   - name: shape_flag
     "@id": "#Object.shape_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Reference band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 115
   - name: shape_xx
     "@id": "#Object.shape_xx"
     datatype: double
     description: Hirata-Seljak-Mandelbaum (HSM) moments. Reference band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 116
   - name: shape_xy
     "@id": "#Object.shape_xy"
     datatype: double
     description: HSM moments. Reference band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 117
   - name: shape_yy
     "@id": "#Object.shape_yy"
     datatype: double
     description: HSM moments. Reference band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 118
   - name: sky_object
     "@id": "#Object.sky_object"
     datatype: boolean
     description: No source was detected here. This object exists to characterize properties of the sky background
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 119
   - name: skymap
     "@id": "#Object.skymap"
     datatype: char
     description: Name of skymap used for coadd projection
     length: 12
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 120
   - name: tract
     "@id": "#Object.tract"
     datatype: long
     description: Skymap tract ID
     fits:tunit:
     ivoa:ucd: meta.id;obs.field
+    tap:principal: 0
+    tap:column_index: 5
   - name: x
     "@id": "#Object.x"
     datatype: double
     description: Centroid from SDSS Centroid algorithm. Reference band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 121
   - name: xErr
     "@id": "#Object.xErr"
     datatype: float
     description: 1-sigma uncertainty on x position. Reference band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 122
   - name: xy_flag
     "@id": "#Object.xy_flag"
     datatype: boolean
     description: General Failure Flag. Reference band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 123
   - name: y
     "@id": "#Object.y"
     datatype: double
     description: Centroid from SDSS Centroid algorithm. Reference band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 124
   - name: yErr
     "@id": "#Object.yErr"
     datatype: float
     description: 1-sigma uncertainty on y position. Reference band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 125
   - name: g_ap03Flux
     "@id": "#Object.g_ap03Flux"
     datatype: double
     description: Flux within 3.0-pixel aperture. Forced on g-band.
     fits:tunit: nJy
+    tap:principal: 0
```

Comment:
The docs for all of these fixed-size circular apertures should note that they are not aperture corrected and hence are not on the same photometric system as our other fluxes; there is some unknown offset (in magnitude space) that we do not fit for.

Oof, and we should really rename these to have sizes in arcseconds.